### ISSUE TITLE
In Example 1, `value` and `lang` are not aliased to `@value` and `@language`

### ISSUE THREAD

**Issue by pchampin** [w3c/vc-data-model#967]
... and therefore, the two schema:name's of the example university are empty objects, and rendered in RDF as blank nodes.
Example 1: 
* [in the spec](https://www.w3.org/TR/vc-data-model/#example-a-simple-example-of-a-verifiable-credential)
* [on the playground](https://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%5B%22https%3A%2F%2Fwww.w3.org%2F2018%2Fcredentials%2Fv1%22%2C%22https%3A%2F%2Fwww.w3.org%2F2018%2Fcredentials%2Fexamples%2Fv1%22%5D%2C%22id%22%3A%22http%3A%2F%2Fexample.edu%2Fcredentials%2F1872%22%2C%22type%22%3A%5B%22VerifiableCredential%22%2C%22AlumniCredential%22%5D%2C%22issuer%22%3A%22https%3A%2F%2Fexample.edu%2Fissuers%2F565049%22%2C%22issuanceDate%22%3A%222010-01-01T19%3A23%3A24Z%22%2C%22credentialSubject%22%3A%7B%22id%22%3A%22did%3Aexample%3Aebfeb1f712ebc6f1c276e12ec21%22%2C%22alumniOf%22%3A%7B%22id%22%3A%22did%3Aexample%3Ac276e12ec21ebfeb1f712ebc6f1%22%2C%22name%22%3A%5B%7B%22value%22%3A%22Example%20University%22%2C%22lang%22%3A%22en%22%7D%2C%7B%22value%22%3A%22Exemple%20d'Universit%C3%A9%22%2C%22lang%22%3A%22fr%22%7D%5D%7D%7D%2C%22proof%22%3A%7B%22type%22%3A%22RsaSignature2018%22%2C%22created%22%3A%222017-06-18T21%3A19%3A10Z%22%2C%22proofPurpose%22%3A%22assertionMethod%22%2C%22verificationMethod%22%3A%22https%3A%2F%2Fexample.edu%2Fissuers%2F565049%23key-1%22%2C%22jws%22%3A%22eyJhbGciOiJSUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..TCYt5XsITJX1CxPCT8yAV-TVkIEq_PbChOMqsLfRoPsnsgw5WEuts01mq-pQy7UJiN5mgRxD-WUcX16dUEMGlv50aqzpqh4Qktb3rk-BuQy72IFLOqV0G_zS245-kronKb78cPN25DGlcTwLtjPAYuNzVBAh4vGHSrQyHUdBBPM%22%7D%7D&frame=%7B%7D&context=%7B%7D)

**Comment by msporny** [comment]
Yes, good catch, we should fix that. We might also consider that we probably don't want to have the first example have a complex language thing in it, so we might want to remove the `lang` and `value` stuff altogether in the examples and only introduce those in a part of the spec that focuses on multilingual issues.

**Comment by OR13** [comment]
I suggest inlining the missing context, and fixing the example via inline context.

**Comment by msporny** [comment]
There are four options here:

1. Alias @language/@value in the base context.
2. Alias @language/@value in the examples context.
3. Don't have any multilingual examples in the spec (which i18n WG is going to have an issue with).
4. Allow lang/value to default to undefined in the base context.

**Comment by OR13** [comment]
```json
{
  
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
   { "@vocab": "https://vendor.example/#", "lang": "@lang"}
  ],
  "id": "http://example.edu/credentials/1872",
  "type": ["VerifiableCredential", "AlumniCredential"],
  "issuer": "https://example.edu/issuers/565049",
  "issuanceDate": "2010-01-01T19:23:24Z",
  "credentialSubject": {
    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
    "alumniOf": {
      "id": "did:example:c276e12ec21ebfeb1f712ebc6f1",
      "name": [{
        "value": "Example University",
        "lang": "en"
      }, {
        "value": "Exemple d'Université",
        "lang": "fr"
      }]
    }
  }
}
```

**Comment by pchampin** [comment]
> There are four options here:
> 
>     1. Alias @language/@value in the base context.

since the base context is protected (IIRC), this could possibly break VCs that currently alias 'lang' and 'value', couldn'it?
> 
>     2. Alias @language/@value in the examples context.
+1
> 
>     3. Don't have any multilingual examples in the spec (which i18n WG is going to have an issue with).
-1
> 
>     4. Allow lang/value to default to undefined in the base context.

I don't get that.

**Comment by pchampin** [comment]
@OR13 your example does not give the expected result (at least, the one that *I* expect when reading the example): the two names are blank nodes with properties `https://vendor.example/#value` and `https://vendor.example/#lang`, while I expected language-tagged literals. Furthermore, `name` does not expand to `http://schema.org/name` anymore, but to `https://vendor.example/#name`.

If we go for a "local patch" of the context, this should be
```json
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://www.w3.org/2018/credentials/examples/v1",
    { "lang": "@language", "value": "@value"}
  ],
  "id": "http://example.edu/credentials/1872",
  "type": ["VerifiableCredential", "AlumniCredential"],
  "issuer": "https://example.edu/issuers/565049",
  "issuanceDate": "2010-01-01T19:23:24Z",
  "credentialSubject": {
    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
    "alumniOf": {
      "id": "did:example:c276e12ec21ebfeb1f712ebc6f1",
      "name": [{
        "value": "Example University",
        "lang": "en"
      }, {
        "value": "Exemple d'Université",
        "lang": "fr"
      }]
    }
  }
}
```

**Comment by OR13** [comment]
@pchampin Thank you, I stand corrected.

**Comment by msporny** [comment]
I'm going to remove the `value` and `lang` properties in example 1 and move it to a multilingual example at the end of the specification. 

This is partially blocked by aliasing value lang in the core context or the examples context. 

There is enough to raise a PR for this.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-04-04-vcwg-special#section1-15) on 2023-04-04

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.15. In Example 1, `value` and `lang` are not aliased to `@value` and `@language` (issue vc-data-model#967)

_See github issue [vc-data-model#967](https://github.com/w3c/vc-data-model/issues/967)._




**Kristina Yasuda:** About Example 1. Manu did you just assign yourself?.  

**Manu Sporny:** Yes.  

</details>

**Comment by msporny** [comment]
PR #1129 has been raised to address this issue. This issue will be closed once PR #1129 is merged.

**Comment by msporny** [comment]
PR #1129 has been merged. Closing.
